pages/agent/v3/monitoring.md

```diff
@@ -0,0 +1,49 @@
+# Monitoring and observing the Buildkite Agent
+
+By default, the Buildkite Agent is only observable either through Buildkite or
+ through log output on the host:
+
+- Job logs, which relate to the jobs the agent runs, are uploaded to Buildkite,
+ and are shown for each step in a build.
+- Agent logs, which relate to how the agent itself is running, are not uploaded
+ or saved (except where this is done by another process such as systemd or
+ launchd).
```

Comment:
Formatting and splitting the sentences:

```suggestion
- **Job logs:** Relate to the jobs the agent runs. These are uploaded to Buildkite and shown for each step in a build.
- **Agent logs:** Relate to how the agent itself is running. These are not uploaded or saved (except where this is done by another process such as systemd or launchd).
```

Reply:
Are there links we could include for systemd and launchd?

Reply:
Thoughts on the formatting update I suggested?

Reply:
Err, yes, forgot to click the button for that one.